SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -121,12 +121,7 @@ WorkerRequirementSet CreateReadACLForAlwaysRelevantEntities()
 {
 	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
```

Comment:
Nit: this is not needed anymore. Can be removed afterwards (the entire function can probably go)